specification/analysisservices/resource-manager/Microsoft.AnalysisServices/2017-07-14/analysisservices.json

```diff
@@ -417,6 +417,39 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AnalysisServices/servers/{serverName}/dissociateGateway": {
```

Comment:
how will calling this API change the payload of the GET /servers/{serverName} existing API that you have? THat has a gatewayResourceId which is associated with the server. What does disassociating that gateway mean?